egg/zoo/dsprites_bvae/data_loaders/data_loaders.py

```diff
@@ -0,0 +1,105 @@
+import torch
+import numpy as np
+import torch.utils.data
+import os
+from torchvision import transforms
+from torch.utils.data import Dataset, DataLoader
+from torch.utils.data.sampler import SubsetRandomSampler
```

Comment:
I wouldn't bet about consistency throughout the entire codebase but I believe we started using [pep8 guidelines](https://www.python.org/dev/peps/pep-0008/#imports) with line-separated groups of import grouped as: 
1. Standard library imports.
2. Related third party imports.
3. Local application/library specific imports.